poe-tasks/get-modified-connectors.sh

```diff
@@ -44,10 +48,20 @@ ignore_patterns=(
 ignore_globs="($(IFS='|'; echo "${ignore_patterns[*]}"))$"
 
 # 3) collect all file changes
-committed=$(git diff --name-only origin/"$DEFAULT_BRANCH"...HEAD)
-staged=$(git diff --cached --name-only)
-unstaged=$(git diff --name-only)
-untracked=$(git ls-files --others --exclude-standard)
+if $PREV_COMMIT; then
```

Comment:
why are we only inspecting the last commit? If a branch has multiple commits, wouldn't we want to inspect the net diff across all of them?

Reply:
good question - this is only for master, where the script is meant to diff the current squashed commit against the previous squashed commit.

otherwise on a PR branch, we do the behaviour you mention and diff everything. This it the else block.

Reply:
aha. that makes sense 👍

Reply:
I'll leave a comment for clarity.

Reply:
Added a comment.